gradle.properties

```diff
@@ -9,6 +9,8 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
+android.enableJetifier=true
```

Comment:
IMHO we can't omit this. Unless the transitive dependency still using Support Library @anangkur

Reply:
alright mas, will remove this, thank you 🙏🏽

Reply:
updated mas @mochadwi please check again thank you..